src/shared/components/palette/case-link/write-case-link-field.html

```diff
@@ -1,16 +1,15 @@
 <ng-container>
-  <div *ngIf="!containsCaseLinkCollection" class="form-group"
-    [ngClass]="{'form-group-error': !caseReferenceControl.valid && (caseReferenceControl.dirty || caseReferenceControl.touched)}">
+  <div *ngIf="!containsCaseLinkCollection" class="form-group">
     <label [for]="id()">
       <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
     </label>
     <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
-    <span class="error-message"
+    <!-- <span class="error-message"
       *ngIf="caseReferenceControl.errors && (caseReferenceControl.dirty || caseReferenceControl.touched)">{{caseReferenceControl.errors
-      | ccdFirstError:caseField.label}}</span>
+      | ccdFirstError:caseField.label}}</span> -->
     <input class="form-control bottom-30" [id]="id()" type="text" [formControl]="caseReferenceControl">
   </div>
 	<div class="form-group" *ngIf="containsCaseLinkCollection">
-		<ccd-write-linked-cases [caseEditPageComponent]="caseEditPageComponent" (onLinkedCasesSelected)="submitLinkedCasses();"></ccd-write-linked-cases>
+		<ccd-write-linked-cases [formGroup]="formGroup" [caseEditPageComponent]="caseEditPageComponent" (onLinkedCasesSelected)="submitLinkedCases();" [isLinkedCasesJourney]="isLinkedCasesEventTrigger()"></ccd-write-linked-cases>
```

Comment:
redudant ; after submitLinkedCases()
if this isLinkedCasesEventTrigger() can be a property rather than method as it defines as property [isLinkedCasesJourney]